controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -119,6 +119,7 @@ func NewKvUiPluginDeplymnt(hc *hcov1beta1.HyperConverged) (*appsv1.Deployment, e
 							},
 						},
 					},
+					NodeSelector:      hc.Spec.Infra.NodePlacement.NodeSelector,
```

Comment:
@tiraboschi this is the change I was talking about for kubevirt-plugin/kubevirt-console-plugin ignoring node placement configuration.

Reply:
can you please do this in a dedicated PR with unit tests and so on?

Reply:
#2319